src/wasm2js.h

```diff
@@ -301,9 +303,10 @@ class Wasm2JSBuilder {
   PassOptions options;
 
   // How many temp vars we need
-  std::vector<size_t> temps; // type => num temps
+  std::unordered_map<Type, Index> temps; // type => num temps
   // Which are currently free to use
-  std::vector<std::vector<IString>> frees; // type => list of free names
+  std::unordered_map<Type, std::vector<IString>>
+    frees; // type => list of free names
```

Comment:
If the comment is placed on its own line, we won't need this odd line break before the variable name.

Reply:
Done.